.travis.yml

```diff
@@ -2,6 +2,7 @@
 language: generic
 
 os: osx
+osx_image: xcode6.1
```

Comment:
Was wondering about this, but it seems to have worked for other people. Will try adding a beta in a new PR.

xref: https://github.com/travis-ci/travis-ci/issues/6522#issuecomment-251766410